extensions/HuggingFace/typescript/hf.ts

```diff
@@ -280,7 +279,7 @@ async function ConstructStreamOutput(
 
     output = {
       output_type: "execute_result",
-      data: delta,
+      data: accumulatedMessage,
```

Comment:
Did this need to be changed? It was already string. It's also fairly easy to handle streamed output chunks from the delta instead of having progressively larger outputs like this. 
We could also add some metadata to denote the final chunk and/or have the final output at the end of the stream which contains the entire accumulated message once all chunks are completed (that's what we do in workbooks)